packages/venia-ui/lib/components/AddressBookPage/addressCard.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { shape, string, bool, arrayOf } from 'prop-types';
+import { Trash2 as TrashIcon, Edit2 as EditIcon } from 'react-feather';
+
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+import Icon from '@magento/venia-ui/lib/components/Icon';
+import defaultClasses from './addressCard.css';
+import LinkButton from '../LinkButton';
+
+const AddressCard = props => {
+    const { address, classes: propClasses } = props;
+
+    const {
+        city,
+        country_code,
+        default_shipping,
+        firstname,
+        lastname,
+        postcode,
+        region: { region },
+        street
+    } = address;
+
+    const streetRows = street.map((row, index) => {
+        return <span key={index}>{row}</span>;
+    });
+
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    const defaultBadge = default_shipping ? (
+        <span className={classes.defaultBadge}>
+            <FormattedMessage
+                id={'addressCard.defaultText'}
+                defaultMessage={'Default'}
+            />
+        </span>
+    ) : null;
+
+    const nameString = `${firstname} ${lastname}`;
+    const additionalAddressString = `${city}, ${region} ${postcode} ${country_code}`;
+
+    return (
+        <div className={classes.root}>
+            <div className={classes.contentContainer}>
+                {defaultBadge}
+                <span className={classes.name}>{nameString}</span>
+                {streetRows}
+                <span>{additionalAddressString}</span>
+            </div>
+            <div className={classes.actionContainer}>
+                <LinkButton
+                    classes={{ root: classes.linkButton }}
+                    onClick={() => console.log('To be completed by PWA-634')}
```

Comment:
I know this is temporary but weirdly I'm not seeing these `console.log`s at all

Reply:
I see them on a local in watch mode, but would expect terser to remove these from prod/PR builds. Was this on the PR deployed instance?

Reply:
Yep, it was. Good call